src/commands/git/branch.ts

```diff
@@ -324,8 +324,16 @@ export class BranchGitCommand extends QuickCommand<State> {
 			}
 
 			QuickCommand.endSteps(state);
+
+			if ((state.reference as GitBranchReference).remote) {
```

Comment:
Here we need to do a check that we are dealing with a branch (because the `reference` could be a commit or tag or something else.